src/testsets.jl

```diff
@@ -186,12 +229,23 @@ function handle_top_level_results!(ts::AbstractTestSet)
         ts.results = AbstractTestSet[]
         ts_nested = ReportingTestSet("Top level tests")
         ts_nested.results = original_results[isa_Result]
+        ts_nested.time_taken = sum(map(x -> x.time_taken, ts_nested.results))
+        ts_nested.start_time = ts.start_time
         push!(ts.results, ts_nested)
         append!(ts.results, original_results[.!isa_Result])
     end
     return ts
 end
 
+"""
+    calc_time_taken!(ts::ReportingTestSet)
+
+Calculate the time taken by subtracting start time (converted to Unix) from now.
+"""
+function calc_time_taken!(ts::ReportingTestSet)
```

Comment:
Is this still used?
I think it isn't

Reply:
Yep it was still being used in `finish`, but as it's now a really simple oneliner I think it makes sense to not break it out into a function